src/server.py

```diff
@@ -416,7 +414,8 @@ def run_seta_details_query():
     date = request.args.get("date", "")
     active = request.args.get("active", 0)
     buildbot = request.args.get("buildbot", 0)
-    branch = request.args.get("branch", '')
```

Comment:
keep in mind that seta runs on fx-team and mozilla-inbound, probably soon to be on autoland.  We will have the same data for all branches.